simple_clock/app.js

```diff
@@ -1,3 +1,115 @@
 window.addEventListener("load", function() {
-  console.log("Hello World!");
+	Clock();//呼叫時鐘、碼表、鬧鐘
+	StopWatch();
+ });
+
+Clock = function(){
+	Clock = document.getElementById('Clock');
```

Comment:
You should not recycle a variable like this, especially `Clock` is visible in global scope.